content/blog/2025/08/04/2025-08-04-update-center-enforce-https.adoc

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Enforced HTTPS for Update Center"
+tags:
+- infrastructure
+- update-center
+- updates.jenkins.io
+- jenkins
+authors:
+- dduportal
+opengraph:
+  image: /images/post-images/2023/01/12/jenkins-newsletter/infrastructure.png
+overview: "Wednesday 06 August 2025, Update Center (`updates.jenkins.io` and `updates.jenkins-ci.org`) will redirect all plain HTTP requests to HTTPS."
+sig:
+  - infra
+---
+
+== Summary (link:https://en.wikipedia.org/wiki/Wikipedia:Too_long;_didn%27t_read[TL;DR])
+
+The service link:https://updates.jenkins.io[https://updates.jenkins.io] will enforce HTTPS protocol the Wednesday 06 August 2025.
+
+All Jenkins users are impacted but should not see any functional change.
+
+Under the hood, any plain-HTTP request made to this service will be redirected to the same location but upgraded to HTTPS scheme.
+
+== What is the "Update Center"?
+
+Jenkins link:https://updates.jenkins.io[Update Center] is a web server at the core of the Jenkins public infrastructure that distributes the plugins, tool installers, and versions index to all Jenkins servers across the world.
+
+From the installation wizard to regular plugin updates, if you run Jenkins, then you use this service under the hood.
+
+Today, it serves around 50 Tb of data (outbound bandwidth) each month from a network of mirrors spread across 3 cloud providers and 2 continents.
+
+Additional information is available in the link:https://github.com/jenkins-infra/helpdesk/issues/4758[GitHub issue].
+
+== What Does It Change for Me?
+
+* Short answer: **Nothing**
+** Since October 2024 with link:https://www.jenkins.io/changelog-old/#v2.480[Jenkins Weekly 2.480] and link:https://www.jenkins.io/changelog-stable/#v2.479.1[Jenkins LTS 2.479.1], Jenkins automatically migrates the Update Center URL.
```

Comment:
```suggestion
** Since October 2024 with link:https://www.jenkins.io/changelog-old/#v2.480[Jenkins Weekly 2.480] and link:https://www.jenkins.io/changelog-stable/#v2.479.1[Jenkins LTS 2.479.1], Jenkins automatically migrates the Update Center URL from HTTP to HTTPS.
```